app/src/main/java/com/example/poste/api/poste/models/User.java

```diff
@@ -67,6 +73,18 @@ public void setPassword(String password) {
         if (!update()) { this.password = oldPassword; }
     }
 
+    /**
+     * Adds a folder to the list of folders this user can access
+     * @param folder the folder to be added to the users list of folders
+     */
+    public void addFolder (Folder folder) {
+        folders.add(folder);
```

Comment:
See above comments re: API calls for these methods

Reply:
What I intend to do is once the method for updating the back end is created, All of these areas of my code that add a folder/post, or remove a folder/post, will then call the method that updates the API. 
Is that method present yet? if so, where is it? I will hock it in as soon as it is available.
(this response is for all the API related comments above, instead of responding to each individually. )

Reply:
We can build out the method, and just change its implementation as necessary.

For instance, you might call user.addFolder(folder); then, we could stub that out to either access the data locally (if testing, I suppose) or make the API call. Really, to avoid duplicate efforts, we should just build out the endpoint and the functionality simultaneously -- that was my understanding of this sprint's goals.

Reply:
I have made the Client side API calls, Please review and let me know if I am missing something.

Reply:
I think what we want here is for User.addFolder to make the API calls necessary to add a Folder. The key isn't adding a folder to the User's ArrayList, its to populate the backend, so the API call is necessary. If we simply add a folder to the user's arraylist, its only contained in memory, not on the backend. It would be more accurate to make a call to the API here to create the Folder and then update the User's folder list from the backend (or, add it to the backend, and simultaneously add it to the Folder's list locally... but this will be overwritten on data update)

Reply:
This is the old User.java file, that i edited before pulling dev. it is no longer needed and this whole java file will be deleted in my next User Story.
we currently have 2 User.java files in 2 different places:
example/poste/api/poste/models/User.java (the old one)
example/poste/models/User.java                  (the new one)

Reply:
Same issue as before, this is the old User.java file. it is soon to be incinerated.